### PR TITLE
Change description and add source in metadata

### PR DESCRIPTION
**Pull request by arturo-seijas**
Metadata changes to adapt to discourse as per https://discourse.charmhub.io/t/changes-to-charmhub-io-pages/9366

### CODE REVIEW

**Review by yanksyoon — APPROVED** [review]
LGTM!

File: metadata.yaml [hunk 1]
```diff
@@ -1,11 +1,21 @@
 name: wordpress-k8s
 display-name: WordPress
 summary: WordPress is an OSS to create a beautiful website.
-description: "WordPress is open source software you can use to create a beautiful website, blog, or app. https://wordpress.org/"
+description: |
+  A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying and managing WordPress on Kubernetes. [WordPress](https://wordpress.com) is the world's most popular website builder, and it's free and open-source.
+  
+  This charm simplifies initial deployment and "day N" operations of WordPress on Kubernetes, including scaling the number of instances, integration with SSO, access to OpenStack Swift object storage for redundant file storage and more.
```
Comment (gtrkiller) [inline metadata.yaml:7]: s/on/in maybe?

Comment (arturo-seijas) [reply metadata.yaml:7]: I'd go with "on"

Comment (gtrkiller) [reply metadata.yaml:7]: makes sense

**Review by gtrkiller — APPROVED** [review]
LGTM

**Pull request merged**